contracts/Helper.sol

```diff
@@ -1,4 +1,6 @@
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity >=0.5.0 <0.7.0;
```

Comment:
space after the `<`

Reply:
You mean changing this line to `pragma solidity >=0.5.0 < 0.7.0;`?
This will compile too, but that's not how semver ranges are usually written ([solidity doc](https://docs.soliditylang.org/en/v0.7.5/structure-of-a-contract.html#state-variables), [semver doc](https://docs.npmjs.com/cli/v6/using-npm/semver#ranges)).

Reply:
ah interesting. I wasn't aware of that. Thanks for the links